go.mod

```diff
@@ -8,5 +8,6 @@ require (
 	github.com/klauspost/compress v1.11.4
 	github.com/spf13/cobra v1.1.3
 	golang.org/x/crypto v0.0.0-20190605123033-f99c8df09eb5
+	nhooyr.io/websocket v1.8.6
```

Comment:
Shouldn’t `github.com/gorilla/websocket v1.4.2 ` be removed from the imports now that this is added?

Reply:
Nope, the tests are still using `gorilla/websocket` for the stub rendezvous server

Reply:
Is there any reason that those couldn't use this as well? Seems better to just use one in my opinion.

Reply:
The idea is we want to validate the new implementation against the existing tests. If the new implementation is correct we shouldn't need to make any adjustments to the tests.

When we are confident everything is correct we can consider switching the test to use the same library.